socorro/processor/general_transform_rules.py

```diff
@@ -4,26 +4,18 @@
 
 from glom import glom
 
-from socorro.lib.transform_rules import Rule
+from socorro.processor.rules.base import Rule
 
 
 class IdentifierRule(Rule):
-    def version(self):
-        return '1.0'
-
-    def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-
-        processed_crash.crash_id = raw_crash.uuid
-        processed_crash.uuid = raw_crash.uuid
-
-        return True
+    def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
+        if 'uuid' in raw_crash:
+            processed_crash['crash_id'] = raw_crash['uuid']
+            processed_crash['uuid'] = raw_crash['uuid']
```

Comment:
The `uuid` gets added by the collector to the raw crash. We should never be processing crashes that don't have a `uuid`, so ... I'm not really sure why we would end up here ever.